knowledge_repo/app/models.py

```diff
@@ -247,6 +261,8 @@ class Post(db.Model):
     keywords = db.Column(db.Text)
     thumbnail = db.Column(db.Text())
 
+    start_publishing = db.Column(db.Integer())
```

Comment:
Are we going for `private` instead of `start_publishing` (or `start_published`)? I think its cleaner.